src/edit/actions/RestoreAction.js

```diff
@@ -1,7 +1,7 @@
 L.RestoreAction = L.EditAction.extend({
   initialize(map, overlay, options) {
-    var mode = L.Utils.getNestedVal(overlay, 'editing', '_mode');
-    var edited = overlay.edited;
+    const mode = L.Utils.getNestedVal(overlay, 'editing', '_mode');
+    const edited = overlay.edited;
```

Comment:
I think this can also change, so we should use `let`! I saw it is set here: https://github.com/publiclab/Leaflet.DistortableImage/blob/00ceec2d7332e39cac1abf0e7ff25ffd11b3e063/src/edit/handles/EditHandle.js#L42

```suggestion
    let edited = overlay.edited;
```

Reply:
Okay so mode and edited are both variables that are liable to change, got it!